knotx-core/knotx-template-engine/src/main/java/com/cognifide/knotx/engine/parser/HtmlParser.java

```diff
@@ -26,14 +26,11 @@
 
 public class HtmlParser {
 
-  private boolean RAW_HTML = false;
-  private boolean RAW_HANDLEBARS_HTML = true;
-
   private static String ANY_TEMPLATE_PATTERN = "(?is).*<script\\s+data-api-type\\s*=\\s*\"templating\".*";
-
   private static Pattern TEMPLATE_PATTERN =
       Pattern.compile("<script\\s+data-api-type\\s*=\\s*\"templating\".+?</script>", Pattern.DOTALL);
-
+  private static boolean RAW_HTML = false;
```

Comment:
Shouldn't those static members be `final`? Is there any case where we want to change them?